mdlight/server.py

```diff
@@ -39,7 +55,7 @@ def parse_args():
     parser.add_argument(
         "--dir",
         metavar="PATH",
-        default=os.curdir,
+        default=os.path.expanduser(os.environ.get("MDLIGHT_DIR", os.curdir)),
```

Comment:
Please, shall we not use environment variables.
As any module, the executable binary has an interface. It is well known that any interface should be the simplest. Arguments are very handy and clear. Easy to understand (`--help`) and easy to use. But ENV is backdoor. Without documentation, you will never know what ENV variables use your tool. It makes your interface tricky and overcomplicated.
Another point is the ENV variables is real global variables. As any global var, it is a pure evil. You could not know what part of code read it and (more important) you could not know what part of code modify it.
I believe we can make software without tricks and cheating.

Reply:
https://github.com/AKindyakov/mdlight/pull/2/files#diff-4b65b12a43bed82a4b3ab662d81ae33fR7
I've noticed this in doc file. Ok, will be removed.